lib/game_manager.rb

```diff
@@ -34,4 +34,76 @@ def lowest_total_score
     end
     min
   end
+
+  def percentage_home_wins
+    home_wins = 0
+    @games.each do |game_id, game|
+      if game.away_goals.to_i < game.home_goals.to_i
+        home_wins += 1
+      end
+    end
+    ((home_wins / @games.length.to_f) * 100).round(2)
+  end
+
+  def percentage_visitor_wins
+    visitor_wins = 0
+    @games.each do |game_id, game|
+      if game.away_goals.to_i > game.home_goals.to_i
+        visitor_wins += 1
+      end
+    end
+    ((visitor_wins / @games.length.to_f) * 100).round(2)
+  end
+
+  def percentage_ties
+    ties = 0
+    @games.each do |game_id, game|
+      if game.away_goals.to_i == game.home_goals.to_i
+        ties += 1
+      end
+    end
+    ((ties / @games.length.to_f) * 100).round(2)
+  end
+
+  def count_of_games_by_season
+    season_data = {}
+    @games.each do |game_id, game|
+      if season_data.include?(game.season)
```

Comment:
potentially update the value of the if statement to check for  nil ? If the key does not exist.